### DISCUSSION TITLE
DDNS Support

### DISCUSSION THREAD

**Discussion by feamsr00** [homeassistant-apps/app-cloudflared#621] [Ideas]
Hello!
Is it possible to use this addon for DDNS, or is that beyond the scope?
(IE updating a host name in CF with the current *public* IP without implementing a tunnel)

**Comment by elcajon** [comment]
Maybe [that's](https://www.home-assistant.io/integrations/cloudflare) what you are looking for?
Otherwise you are right, DDNS is not in the scope of this addon, sorry.
